src/language-js/comments.js

```diff
@@ -950,12 +959,32 @@ function getCommentChildNodes(node, options) {
   }
 }
 
+function removeJsDocAsterisks(commentValue) {
```

Comment:
I can't find a test case that actually requires this. I think the intention is to exclude leading `  *` after a newline after the `@type` annotation? If so, I think it'd be plenty fine to just search for `@type` and not care about having a non-whitespace char.

Reply:
Agree. This is leftover code from the check for curly braces around the type.